docs/reference/karmada-api/work-resources/resource-binding-v1alpha2.md

```diff
@@ -546,6 +546,16 @@ ResourceBindingSpec represents the expectation of ResourceBinding.
 
     Namespace represents the namespace of the Binding. It is required for ResourceBinding. If Namespace is not specified, means the referencing is ClusterResourceBinding.
 
+- **rescheduleTriggeredAt** (Time)
+
+  RescheduleTriggeredAt is a timestamp representing when the referenced resource is triggered rescheduling. When this field is updated, it means a rescheduling is manually triggered by user, and the expected behavior of this action is to do a complete recalculation without referring to last scheduling results. It works with the status.lastScheduledTime field, and only when this timestamp is later than timestamp in status.lastScheduledTime will the rescheduling actually execute, otherwise, ignored.
```

Comment:
```suggestion
  rescheduleTriggeredAt is a timestamp representing when the referenced resource is triggered rescheduling. When this field is updated, it means a rescheduling is manually triggered by user, and the expected behavior of this action is to do a complete recalculation without referring to last scheduling results. It works with the status.lastScheduledTime field, and only when this timestamp is later than timestamp in status.lastScheduledTime will the rescheduling actually execute, otherwise, ignored.
```

Reply:
The English documentation is automatically generated via `reference-docs`. In fact, the [kubernetes API documentation](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/) is in the same style.

Reply:
ok

Reply:
Cool, seems we have made an agreement now.
/lgtm
/approve